app/controllers/shows_controller.rb

```diff
@@ -64,20 +64,21 @@ def destroy
   # migrating from protected attr to strong param
   # standard found here: https://www.fastruby.io/blog/rails/upgrades/strong-parameters-migration-guide.html
   private
+
   def season_new_params
     params.permit :season
     params.fetch :season, Time.this_season
   end
 
   def show_params
     permitted = params.require(:show).permit(:name, :event_type,
-                                             :season,
```

Comment:
this shows the danger of not rebasing often.  in your original code, you had forgotten to add :season to the params list, which caused a customer problem.  i fixed it and merged the fix rapidly to main, but you didn't rebase against my main, so your PR is going to reintroduce the bug, which i'll have to fix manually.